wunderio/next4drupal-project
# Feature/nex 70 security headers

*[Link to ticket: NEX-70](https://wunder.atlassian.net/browse/NEX-70)*

*Changes proposed in this PR:*
- Implement more strict CSP header

*How to test:*
1. Check observatory score: https://observatory.mozilla.org/analyze/feature-nex-70-security-headercc9.next4drupal-project.dev.wdr.io 
2. Check the site and verify all the resources load and requests work properly 

*Link to feature environment:*
https://feature-nex-70-security-headercc9.next4drupal-project.dev.wdr.io/
